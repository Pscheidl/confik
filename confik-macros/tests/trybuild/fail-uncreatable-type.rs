#[derive(confik::Configuration, Debug)]
enum C {}

fn main() {}
