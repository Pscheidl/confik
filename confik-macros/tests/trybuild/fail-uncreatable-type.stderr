error: Cannot create a builder for a type that cannot be instantiated: C
 --> tests/trybuild/fail-uncreatable-type.rs:2:6
  |
2 | enum C {}
  |      ^
